lib/ansible/plugins/action/__init__.py

```diff
@@ -659,7 +659,7 @@ def _fixup_perms2(self, remote_paths, remote_user=None, execute=True):
                         'member. In this situation, '
                         'allow_world_readable_tmpfiles is a no-op. See this '
                         'URL for more details: '
-                        'https://docs.ansible.com/ansible/become.html'
+                        'https://docs.ansible.com/ansible/latest/user_guide/become.html'
```

Comment:
Could you please add a changelog entry for this change [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) so that we can backport this change?

Reply:
should use `get_versioned_doclink` function as 'latest' might not apply to the 'current version'